scripts/vagrant/Vagrantfile

```diff
@@ -47,12 +47,14 @@ Vagrant.configure("2") do |config|
     master.vm.hostname = "kube-master"
     master.vm.provision "shell", path: "scripts/configureK8smaster.sh"
     master.vm.provision "shell", path: "scripts/coredump.sh"
+    master.vm.network "private_network", ip: "fdf8:b425:ec6e:aa6b:0:0:0:1"
```

Comment:
Shall we use an address form the private IPv6 range :`fd00::/8`?

Reply:
I believe this one is also part of a private IPv6 address range. In practice, everything starting with `fd` can be considered as private or ULA in the context of IPv6.

Ref: [link](https://simpledns.com/private-ipv6)